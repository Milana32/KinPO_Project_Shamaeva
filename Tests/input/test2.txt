11 _- 3 4 + 7 *  > 