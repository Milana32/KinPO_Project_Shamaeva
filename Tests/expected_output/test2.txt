(_- 11) > (3 + 4) * 7
(3 + 4) * 7 < (_- 11)